images/scripts/lib/build-deps.sh

```diff
@@ -46,7 +47,8 @@ esac
 
 # pull nodejs-devel on Fedora for compliance with the guidelines on using nodejs modules:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Node.js/#_buildrequires
-case "$OS_VER" in
+case "$OS_VARIANT" in
```

Comment:
Ah, that's the "pass twice" bit. But please keep the VERsion here as well, we needed it in the past. I suggest to pass a single "$ID $VERSION_ID $VARIANT_ID" argument and match on that.